project/apps/matching-service/src/constants/websocket.ts

```diff
@@ -0,0 +1,5 @@
+export const SOCKET_USER_KEY = 'socket-user';
+export const USER_SOCKET_KEY = 'user-socket';
+
+export const WEBSOCKET_RETRY_ATTEMPTS = 3;
```

Comment:
a bit of a personal opinion here but i think we can afford to increase the number of retry attempts.

at retry_attempts = 3 it'll only attempt to connect for 2 seconds:
attempt 1: 0th second
attempt 2: 0th second (sent immediately after based on `WEBSOCKET_RETRY_DELAY * 2 ** (attempt - 1)`)
attempt 3: 2nd second

perhaps can increase to 5?

Reply:
Not sure if i miswrote it but based on the code it is actually on exponential backoff (1, 2, 4) = 7 seconds in total. I can perhaps increase by 1 more to 4 attemps making the fourth one 8 seconds

Reply:
edit: yikes, just tested it for sanity check, missing parenthesis